### DISCUSSION TITLE
how can we add alpinejs in Angular project properly without using cdn?

### DISCUSSION THREAD

**Discussion by cadentic** [alpinejs/alpine#4241] [1. Help]
how can we add alpinejs in Angular project properly without using cdn? 
shall I go with npm I alpinejs or there is any angular specific packages ?

**Comment by SimoTod** [comment]
No angular specific version. You can just use npm: https://alpinejs.dev/essentials/installation#as-a-module

**Reply by cadentic** [reply]
we may add those cdn still it will not support all the syntax or directives for alpine in angular 17 / angular 18

**Comment by ekwoka** [comment]
For what reason would you want to ad Alpine to an Angular project?

**Reply by cadentic** [reply]
avoiding tons of refactoring while using ui with tailwindjs

**Reply by ekwoka** [reply]
Tailwind UI?

I think it will be more work to try to use Alpine inside angular than just changing the templates to be angular.
